garak/generators/base.py

```diff
@@ -74,11 +76,17 @@ def generate(self, prompt: str) -> List[str]:
 
         self._pre_generate_hook()
 
+        if generations_this_call == -1:
+            generations_this_call = self.generations
+
+        elif generations_this_call == 0:
```

Comment:
Do we want to warn users in this case? Seems like if someone calls a generator with no generations, they've probably made a mistake.

Reply:
agree, added logging